### DISCUSSION TITLE
Unhandled status from server when sending some code chunks

### DISCUSSION THREAD

**Discussion by BWPearson** [community/community#56076] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hi, I've just started using Copilot Chat and I'm regularly running into this error when asking for explanations and simplifications on Typescript components.

`Unhandled status from server: 400 Bad request: bad request: role must be "user", got "assistant"`

Has anyone run into it before or got a solution? I'd be quick to blame config on my end being a new user but I'd love a second opinion

**Comment by d3vilh** [comment]
as per https://github.com/microsoft/vscode/issues/182359  it is new limitation in chat token size.

Previously, however it worked just fine. 
Hope this will be fixed soon, as coping large parts of my code for review and suggestions was the main CopilotX/chat feature for me and the actual reason for subscription.

Free version of ChatGPT don't have such limitation thou :)
